v2/utils.go

```diff
@@ -0,0 +1,35 @@
+package jsonrpc
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+var (
```

Comment:
they are not constant. they can be adjusted at run time as well as compile time. and no these are in the place where they are relevant.